uktrade/data-hub-frontend
# feature/CLS2-643-select-investment-project

## Description of change

Allow a user to choose an investment project for a task, instead of only using the value in the url

## Test instructions

Go to http://localhost:3000/tasks/create and choose a company, the investment project field will appear

## Screenshots

### Before

![image](https://github.com/uktrade/data-hub-frontend/assets/102232401/a039bd16-450e-452d-bf02-39e3e37ea41f)

### After

![image](https://github.com/uktrade/data-hub-frontend/assets/102232401/63b769b0-8e0c-4c73-95d2-2e11e06221b6)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
